sigmahq/sigmahq_conventions.md

```diff
@@ -67,10 +71,22 @@ title: Suspicious Office Child Process
 
 All newly created rules must start with a status of `experimental`
 
+## Description
+
+- All rule descriptions must start with the word `Detects`
```

Comment:
```suggestion
- All rule descriptions must explain what the rule detects. A best practice therefore is to start with the word `Detects`
```
I think there can be good descriptions that don't start with "detects". It's just easier to use the word to guide the thinking process.